test/libsolidity/semanticTests/isoltestTesting/format_raw_string_with_control_chars.sol

```diff
@@ -0,0 +1,12 @@
+contract C {
+    function f(string memory s) external pure returns (string memory) {
+        return s;
+    }
+}
+// NOTE: The test is here to illustrate the problem with formatting control chars in strings in
+// test expectations but unfortunately it can only be triggered manually. It does not test anything
+// unless you introduce a difference in expectations to force isoltest to reformat them.
+// ====
+// compileViaYul: also
+// ----
+// f(string): 0x20, 16, "\xf0\x9f\x98\x83\xf0\x9f\x98\x83\xf0\x9f\x98\x83\xf0\x9f\x98\x83" -> 0x20, 16, "\xf0\x9f\x98\x83\xf0\x9f\x98\x83\xf0\x9f\x98\x83\xf0\x9f\x98\x83" # Input/Output: "😃😃😃😃" #
```

Comment:
Not sure if it's worth keeping this test. To trigger it you have to change expectations, e.g. replace `16` with `15`.

`strings.sol` actually has a higher chance of triggering it by accident because it has gas expectations, which change often.